server.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('public'));
+
+app.get('/', (req, res) => {
+  res.render('index.html');
+});
+
```

Comment:
What happens if I directly enter `/tasks` in my browser?

Reply:
Or whatever is the path for the task creation form

Reply:
For task creation it's **/task/index.html** and we directly navigate to task creation page 
https://test-website-dashboard.herokuapp.com/task/index.html